Source/WebKit/NetworkProcess/storage/CacheStorageDiskStore.cpp

```diff
@@ -318,12 +317,14 @@ std::optional<CacheStorageRecord> CacheStorageDiskStore::readRecordFromFileData(
         if (storedInfo->metaData.bodyHash != computeSHA1(bodyData, m_salt))
             return std::nullopt;
 
+        // FIXME: avoid copying inline body data here, perhaps by adding offset support to
```

Comment:
There is still an extra dirty memory copy here, but hopefully the impact is minimal because it only impacts small response bodies where the response data is stored in-line with the cached request metadata.